pytripgui/field_editor/field_model.py

```diff
@@ -0,0 +1,85 @@
+from pytrip.tripexecuter import Field
+
+import logging
+logger = logging.getLogger(__name__)
+
+
+class FieldModel(object):
+    def __init__(self, field=None):
+        if field is None:
+            self.field = Field()
+        else:
+            self.field = field
+
+        self.field_test()
+
+    def field_test(self):
+        field = self.field
+
+        lenth_of_isocenter = len(field.isocenter)
+        if lenth_of_isocenter is not 0 and lenth_of_isocenter is not 3:
+            raise Exception("Field.isocenter length is({}) instead of 0 or 3".format(lenth_of_isocenter))
+
+    def set_isocenter_manually(self, bool):
+        if bool is True:
+            self.field.isocenter = [0.0, 0.0, 0.0]
+        else:
+            self.field.isocenter = []
+
+    def is_isocenter_manually(self):
+        if len(self.field.isocenter) is 3:
+            return True
+
+        return False
+
+    def get_isocenter_values(self):
+        if self.is_isocenter_manually() is False:
+            raise Exception("isocenter will be set automatically")
+        return self.field.isocenter
+
+    def set_isocenter_values(self, isocenter):
+        if self.is_isocenter_manually() is False:
+            raise Exception("isocenter will be set automatically")
+        self.field.isocenter = isocenter
+
+    def get_gantry_angle_value(self):
+        return self.field.gantry
+
+    def set_gantry_angle_value(self, gantry_angle):
+        self.field.gantry = gantry_angle
+
+    def get_couch_angle_value(self):
```

Comment:
Do we really need getters and setters like in Java ? See https://stackoverflow.com/a/36943813

Reply:
ok then. Is there a sens to create a model class which inherits pytrip/executor/field, or should I just take pytrip/executor/field as a model?

Reply:
I think it doesn't make sense to create a separate class now. Another question is more important: how to make a MVC architecture  involving:
 - plan, field, etc... (model)
 - all necessary views needed to visualise data, run and monitor trip simulations
 - necessary controllers to bind everything together

Such architecture require multiple classes which are interconnected (i.e. part of model responsible for CT data being used by many controllers/views).